src/app/app.module.ts

```diff
@@ -154,6 +154,7 @@ import {
 import {
     LayerStatisticsTextualDetailsComponent
 } from './plots/layer-statistics-view/layer-statistics-textual-details/layer-statistics-textual-details.component';
+import {CreateRgbComponent} from './operators/dialogs/create-rgb/create-rgb.component';
```

Comment:
Since we already do visualization of single layers with RGB colors my suggestion is to change this new 'rgb' naming to 'Composite' or 'LayerComposite'

Reply:
see https://github.com/umr-dbs/wave/pull/12/commits/728ca63d51771dc5b52bd260da6c4c402a6b3b50